include/service/entities/password_reset_service.js

```diff
@@ -96,15 +96,15 @@ module.exports = function(pb) {
             }
 
             //attempt to retrieve any existing reset
-            self.getSingle({where: {userId: userObj.id}}, function(err, passwordResetObj) {
+            self.getSingle({where: {user_id: userObj[pb.DAO.getIdField()].toString()}}, function(err, passwordResetObj) {
```

Comment:
Oh snap, and I thought it was the other way round. Welp, this PR needs an overhaul then, seems like I was looking in the wrong place.